src/sdk/conference/src/main/java/owt/conference/ConferenceClient.java

```diff
@@ -21,9 +21,13 @@
 import java.util.Collections;
```

Comment:
#94 has been fixed by #123. My idea to make everything ready before firing success callback.

Reply:
Ok, I remove these change.